build_wheel.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+#************************************************#
+#           build_wheel.sh                       #
+#    Build a Python3 wheel for YOLACT            #
+#************************************************#
+
+REPO_NAME="yolact"
+NEEDED_FOLDERS=("data" "external" "layers" "scripts" "utils" "web")
+REQUIREMENTS_FILE="requirements.txt"
+SETUP_FILE="setup.py"
+PYTHON_EXTENSION=".py"
+PACKAGE_TEST_DIR="/tmp/$USER/pip_package_test"
+
+# --------------------------------------------------------- #
+# main ()                                                   #
+# drives the entire build process                           #
+# Returns: 0 on success, non-zero if something goes wrong.  #
+# --------------------------------------------------------- #
+main() {
+  echo "Starting build ..."
+  pre_build_setup &&
+    build &&
```

Comment:
I mentioned the `set -e` stuff earlier because if you need to add more logic to this flow and can't just chain `&&`s together, `set -e` would help you to ensure that the script exited immediately upon hitting an error.